BioChem/UsersGuide/References.mo

```diff
@@ -3,5 +3,5 @@ within BioChem.UsersGuide;
+    Documentation(info = "<html><head></head><body><h2>References</h2><p><b>BioChem - A Biological and Chemical Library for Modelica&nbsp;</b><br>Emma Larsdotter Nilsson, Peter Fritzson. Proceedings of the 3rd International Modelica Conference (November 3-4, Linköping, Sweden) 2003, pp. 215-220&nbsp;<b>(<a href=\"https://www.researchgate.net/publication/237321637_BioChem_-_A_Biological_and_Chemical_Library_for_Modelica\">link</a>)</b><br><br><b>A minimal cascade model for the mitotic oscillator involving cyclin and cdc2 kinase&nbsp;</b><br>Albert Goldbeter. Proc. Nati. Acad. Sci. USA Vol. 88, pp. 9107-9111, October 1991 Cell Biology (<a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC52661/\">link</a>)&nbsp;<br><br><b>Hierarchical modeling of diabetes</b><br>Elin Nyman. ISRN:LiU-IKE-EX-09/14. Linköping University 2009 (<a href=\"http://liu.diva-portal.org/smash/record.jsf?pid=diva2%3A231708&amp;dswid=mainwindow\">link</a>)<br><br><b>A Minimal Generic Model of Bacteria-Induced Intracellular Ca<sup>2+</sup>&nbsp;Oscillations in Epithelial Cells</b><br>Camilla Oxhamre, Agneta Richter-Dahlfors, Vladimir P. Zhdanov, and Bengt Kasemo. Biophysical Journal Volume 88 April 2005 2976-2981 (<a href=\"https://www.sciencedirect.com/science/article/pii/S000634950573349X\">link</a>)<br><br><b>A mathematical model of metabolic insulin signaling.</b><br>A. Sedaghat, R, A. Sherman, and J. Quon, Michael. American Journal of Physiology - Endocrinology and Metabolism, 283:1048-1101, Jul 2002 (<a href=\"https://pubmed.ncbi.nlm.nih.gov/12376338/\">link</a>)<br><br><b>Modeling the cell division cycle: cdc2 and cyclin interactions&nbsp;</b><br>John J. Tyson. Proc. Nati. Acad. Sci. USA Vol. 88, pp. 7328-7332, August 1991 Cell Biology&nbsp;<b>(<a href=\"https://pubmed.ncbi.nlm.nih.gov/1831270/\">link</a>)</b><br><br><b>Modeling Feedback Loops of the Mammalian Circadian Oscillator&nbsp;</b><br>Becker-Weimann S, Wolf J, Herzel H, Kramer A. Biophysical Journal Volume 87 November 2004 3023-3034 (<a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1304775/\">link</a>)</p></body></html>"));
```

Comment:
researchgate? Why not the source? https://www.modelica.org/events/Conference2003/papers/h21_Larsdotter_Nilsson.pdf
And http://urn.kb.se/resolve?urn=urn:nbn:se:liu:diva-19076 is the "permanent link" according to the second URL

Reply:
Done